src/main/java/net/minecraftforge/fml/common/FMLCommonHandler.java

```diff
@@ -202,7 +202,7 @@ public Side getSide()
     public Side getEffectiveSide()
     {
         Thread thr = Thread.currentThread();
-        if (thr.getName().equals("Server thread") || thr.getName().startsWith("Netty Server IO"))
+        if (thr.getName().equals("Server thread") || thr.getName().startsWith("Netty Epoll Server IO") || thr.getName().startsWith("Netty Server IO"))
```

Comment:
No, that might lead to things being detected as the Minecraft server that are some other kind of server